scripts/fix_embargo_approved_logs.py

```diff
@@ -26,14 +26,18 @@ def get_targets():
 
 
 def fix_embargo_approved_logs(targets):
+    count = 0
     for log in targets:
         node_id = log.params['node']
         node = Node.load(node_id)
         if node.is_registration:
+            log_params = log.params
             log.params['node'] = node.registered_from_id
             log.params['registration'] = node._id
+            logger.info('Updating params of log {} from {} to {}'.format(log._id, log_params, log.params))
```

Comment:
Since log_params just points to `log.params`, they will be logged as identical. You'd need to copy the original params to fix this. I can make this change when I merge.